tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1334,6 +1336,7 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      conditions:
         - "asic_type in ['cisco-8000']"
         - "topo_type in ['m0', 'mx']"
+        - "asic_type in ['vs']"
```

Comment:
pfcwd/test_pfc_config.py is included in PR test and running good, no need to skip